src/components/InterestSignupForm.js

```diff
@@ -33,11 +36,27 @@ class InterestSignupForm extends Component {
     render () {
         let canSubmit = (this.state.email.length > 3);
         return (
-            <div>
```

Comment:
in the spirit of reusable components, it'd be good to keep `InterestForm` as just the form itself, (i.e. `return(<div className="interestForm">...etc`, since once we open applications we'll cease to use this component, but we'll still want the logo/name/location/date copy